packages/story-editor/src/app/media/local/useContextValueProvider.js

```diff
@@ -205,7 +212,25 @@ export default function useContextValueProvider(reducerState, reducerActions) {
         removeBaseColorProcessing({ id });
       })();
     },
-    [setBaseColorProcessing, removeBaseColorProcessing, updateBaseColor]
+    [setBaseColorProcessing, updateBaseColor, removeBaseColorProcessing]
+  );
+
+  const processMediaBlurhash = useCallback(
+    (resource) => {
+      const { blurHashProcessed, blurHashProcessing } = stateRef.current;
+      const { id } = resource;
+
+      (async () => {
+        // Simple way to prevent double-uploading.
+        if (blurHashProcessed.includes(id) || blurHashProcessing.includes(id)) {
```

Comment:
I tweaked this, to only generate on one media element, still didn't help the issue.